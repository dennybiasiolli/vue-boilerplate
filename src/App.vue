<template>
  <v-app>
    <Nav />
    <Toolbar />
    <v-content>
      <v-container fluid>
        <router-view />
      </v-container>
    </v-content>
    <v-footer app>
      <v-spacer />
      &copy; {{ year }} - Denny Biasiolli
      <v-spacer />
    </v-footer>
  </v-app>
</template>

<script>
import Nav from './components/Nav'
import Toolbar from './components/Toolbar'

export default {
  name: 'App',
  components: {
    Toolbar,
    Nav
  },
  computed: Object.assign({
    year: () => new Date().getFullYear()
  })
}
</script>

<style></style>
