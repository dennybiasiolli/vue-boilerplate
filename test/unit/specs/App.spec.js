import { shallow } from 'vue-test-utils'
import App from '@/App'

describe('App', () => {
  test('should match snapshot', () => {
    const component = shallow(App)
    expect(component.element).toMatchSnapshot()
  })

  // Evaluate the results of functions in
  // the raw component options
  test('sets the correct components data', () => {
    expect(App.name).toBe('App')
  })
})
