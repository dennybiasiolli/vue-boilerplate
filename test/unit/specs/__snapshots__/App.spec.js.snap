// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div
  class="application theme--light"
  data-app="true"
  id="app"
>
  <div
    class="application--wrap"
  >
    <!---->
     
    <!---->
     
    <main
      class="content"
      style="padding: 0px 0px 0px 0px;"
    >
      <div
        class="content--wrap"
      >
        <div
          class="container fluid"
        >
          <!---->
        </div>
      </div>
    </main>
     
    <footer
      class="footer footer--fixed"
      style="height: 32px;"
    >
      <div
        class="spacer"
      />
      
    Â© 2018 - Denny Biasiolli
    
      <div
        class="spacer"
      />
    </footer>
  </div>
</div>
`;
